telegram/client.py

```diff
@@ -21,6 +22,8 @@
 
 MESSAGE_HANDLER_TYPE: str = 'updateNewMessage'
 
+AuthorizationState = enum.Enum('AuthorizationState', 'IDLE WAIT_CODE WAIT_PWD DONE FAILED')
```

Comment:
> I am not aware that enums names should conform to UPPER_CASE naming style.
```
telegram/client.py:25:0: C0103: Constant name "AuthorizationState" doesn't conform to UPPER_CASE naming style (invalid-name)
```

It is a convention for constants and in your case, the linter just sees it as a constant. You need to change it to something like this, and it can be reused in the login method:

```python
class AuthorizationState(enum.Enum):
    NOT_STARTED: str = None
    WAIT_TDLIB_PARAMETERS: str = 'authorizationStateWaitTdlibParameters'
    WAIT_ENCRYPTION_KEY: str = 'authorizationStateWaitEncryptionKey'
    WAIT_PHONE_NUMBER: str = 'authorizationStateWaitPhoneNumber'
    WAIT_CODE: str = 'authorizationStateWaitCode'
    WAIT_PASSWORD: str = 'authorizationStateWaitPassword'
    READY: str = 'authorizationStateReady'
```

Reply:
Roger that